src/OpenTelemetry/Internal/OpenTelemetrySdkEventSource.cs

```diff
@@ -415,6 +424,36 @@ public void InvalidEnvironmentVariable(string key, string value)
             this.WriteEvent(47, key, value);
         }
 
+        [Event(48, Message = "OpenTelemetry IHostedService registered in application services.", Level = EventLevel.Informational)]
+        public void HostedServiceRegistered()
+        {
+            this.WriteEvent(48);
+        }
+
+        [Event(49, Message = "OpenTelemetry IHostedService application services registration skipped. Reason: '{0}'", Level = EventLevel.Informational)]
```

Comment:
Warning? This is user-actionable right? We could point users to the steps they should take to resolve this.

Reply:
There are 3 reasons we could log skipped:

1) `RuntimeFeature.IsDynamicCodeSupported` == `false`
2) `Microsoft.Extensions.Hosting.IHostedService, Microsoft.Extensions.Hosting.Abstractions` type could not be found
3) Reflection/dynamic code blew up. This will also cause an Error to be written for `HostedServiceRegistrationFailure` event.

1 + 2 could be normal/expected, just depends on who is setting up the host. I just updated it to be a warning but if any of that changed your mind LMK.